satellite/satellitedb/migrate.go

```diff
@@ -927,6 +927,13 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 					`UPDATE nodes SET disqualified=NULL WHERE disqualified IS NOT NULL AND audit_reputation_alpha / (audit_reputation_alpha + audit_reputation_beta) >= 0.6;`,
 				},
 			},
+			{
+				Description: "Add pending audit path",
+				Version:     40,
+				Action: migrate.SQL{
+					`ALTER TABLE pending_audits ADD COLUMN path text NOT NULL DEFAULT '';`,
```

Comment:
should we clear the table as part of the migration?

Reply:
I don't think we have to now that I added checks for empty paths

Reply:
I'd vote for clearing the table and removing the check for empty paths. This way we simplify the production code a little bit, which is a good thing in the long-term. Who will remember why we have this check for empty paths in a few months?

If we go in the direction of clearing the `pending_audits` table, we should also set all `containment` values to `false` in the `nodes` table.

Reply:
will fix